bliss/sleep.py

```diff
@@ -239,7 +239,7 @@ def _get_loss(self, batch):
         var_params = self.image_encoder.encode(images, background)
         var_params_flat = rearrange(var_params, "b nth ntw d -> (b nth ntw) d")
```

Comment:
This could be refactored to avoid using the now-deprecated `encode_for_n_sources_flat`

Reply:
Fixed this